popup.html

```diff
@@ -41,6 +41,7 @@ <h4 id="selectedTheme"></h4>
         <option value="cm-s-eclipse">Eclipse</option>
         <option value="cm-s-elegant">Elegant</option>
         <option value="cm-s-erlang-dark">Erlang Dark</option>
+        <option value="cm-s-google-doc-light">Google Doc Light</option>
```

Comment:
I really like this idea. Can we move this as the **first** theme, I know I had everything alphabetical, but this seems to be useful for folks preparing for Google.  Also if possible, can we name it like "Google Interview" or something catchy 😄

Reply:
Sure! I'm glad you like it. 😁 

I am little busy handling some errands but I will get it done as soon as possible. I am also planning on creating a "Google Interview - Night" style also. 

Action Items:

- [ ] Move "Google Interview" __and__ "Google Interview" Dark to top of dropdown list
- [ ] Add a "Google Interview - Night" style

Reply:
Great! Take your time!

Reply:
Hey Vishnu, hope all is well.

So, I just started again going through and implementing the changes. However, I noticed in my initial implementation of **Google Interview Day**, the CSS style wrapper via Code-Mirror is _not_ being generated.

Would you happen to know why? Its been puzzling me for a while now.

Reply:
I guess you got the issue, there were some hidden 3024 classnames 😁

Reply:
Yep, my bad. 🤦🏼 

Was going crazy until I realized it!